snap/squashfs/squashfs.go

```diff
@@ -202,7 +202,7 @@ func (s *Snap) Unpack(src, dstDir string) error {
 	usw := newUnsquashfsStderrWriter()
 
 	var output bytes.Buffer
-	cmd := exec.Command("unsquashfs", "-n", "-f", "-d", dstDir, s.path, src)
+	cmd := exec.Command("unsquashfs", "-u", "-n", "-f", "-d", dstDir, s.path, src)
```

Comment:
what does -u do? I don't see it in the manpage or --help output

Reply:
It's an alias for user attrs. Let me find the source: https://github.com/plougher/squashfs-tools/blob/master/squashfs-tools/unsquashfs.c#L4268